src/Security/AuthSamples/samples/ClaimsTransformation/ClaimsTransformation.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
```

Comment:
Nit: you can remove this - it's already the default for projects under 'samples' or testassets.